sparse/core.py

```diff
@@ -1366,7 +1375,7 @@ def concatenate(arrays, axis=0):
 
 def stack(arrays, axis=0):
     assert len(set(x.shape for x in arrays)) == 1
-    arrays = [x if type(x) is COO else COO(x) for x in arrays]
+    arrays = [COO(x) for x in arrays]
```

Comment:
I would leave these as-is, or maybe change them to `arrays = [x if isinstance(x, COO) else COO(x) for x in arrays]`. Generally, we want to avoid making copies where possible.

I realize what you are doing is a shallow copy, not a deep one, but in the future, if we allow manipulation of arrays (if we make the `COO` type mutable), we might need to change this to a deep copy, and that might be expensive.